compiler/src/iree/compiler/API/Internal/IREEGPUDialectCAPI.cpp

```diff
@@ -420,5 +424,111 @@ ireeHALExecutableTargetAttrGetGPUTargetInfo(MlirAttribute attr) {
       wgpAttr.getMaxThreadCountPerWorkgroup();
   targetInfo.maxWorkgroupMemoryBytes = wgpAttr.getMaxWorkgroupMemoryBytes();
 
+  targetInfo.mmaIntrinsics = wrap(builder.getArrayAttr({}));
+  mlir::iree_compiler::IREE::GPU::MMAOpsArrayAttr mmaOpsArray =
+      wgpAttr.getMma();
+  if (mmaOpsArray) {
```

Comment:
https://llvm.org/docs/CodingStandards.html#use-early-exits-and-continue-to-simplify-code